src/data/candidates/devora-davis.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Devora \"Dev\" Davis",
```

Comment:
These names will need to exactly match the names from the backend (or we will need a separate ID field which has the names exactly as they are stored on the backend).

The spreadsheet for reference is here: https://docs.google.com/spreadsheets/d/1OuFAqp2ir5I-p19ku8hsK7fHRN61P-UnofeSCBh1aPI/edit#gid=1886472303

Names that will need changing (either as part of this PR or as a follow-up: "Dev Davis" and "Jacob "Jake" Tonkel"

Reply:
@rwalek668 Yeah, I think an id field will probably make the most sense. It looks like @geleazar1000111 is adding one in the api, so I'll probably just add those into these json files. Thanks for flagging those!